src/pages/Main/Filter/Filter.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import './Filter.scss';
+import COLOR_LIST from './colorData';
+
+const Filter = () => {
+  const [priceValue, setPriceValue] = useState(50000);
+  const handlePrice = e => {
+    setPriceValue(e.target.value);
+  };
+
+  return (
+    <div className="filter">
+      <div className="filterInner">
+        {/* gender */}
+        <div className="productGender filterLayout">
+          <h3>성별</h3>
+          <div className="womenCheck">
+            <input type="checkbox" id="checkBox" />
+            <label for="checkBox" />
+            <span>여성</span>
+          </div>
+          <div className="menCheck">
+            <input type="checkbox" id="checkBox2" />
+            <label for="checkBox2" />
+            <span>남성</span>
+          </div>
```

Comment:
```suggestion
 <div className="checkBoxList">
            <input type="checkbox" id="checkWomen" className="checkBoxInput" />
            <label htmlFor="checkWomen" />
            <span className="selectTitle">여성</span>
          </div>
          <div className="checkBoxList">
            <input type="checkbox" id="checkMen" className="checkBoxInput" />
            <label htmlFor="checkMen" />
            <span className="selectTitle">남성</span>
          </div>
```

동일한 구조이기 때문에 className을 나눠서 하지 않고 동일하게 구현해도 되겠습니다!